src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -961,6 +1002,11 @@ public DistributedApplicationModel(System.Collections.Generic.IEnumerable<IResou
         public IResourceCollection Resources { get { throw null; } }
     }
 
+    public static partial class DistributedApplicationModelExtensions
+    {
+        public static System.Collections.Generic.IEnumerable<IResource> GetComputeResources(this DistributedApplicationModel model) { throw null; }
```

Comment:
`IComputeResource`?

Reply:
It can't be `IComputeResource` since it will return resources that only have the ContainerImageAnnotation applied to it. The actual .NET object might not implement IComputeResource.